bleak/backends/bluezdbus/manager.py

```diff
@@ -472,8 +472,26 @@ async def passive_scan(
             )
             self._device_removed_callbacks.append(device_removed_callback_and_state)
 
+            # Once a monitoring job is activated by BlueZ, the client can expect to get notified
+            # on the targeted advertisements no matter if there is an ongoing discovery session
+            # (started/stopped with org.bluez.Adapter1.StartDiscovery/StopDiscovery, as done when
+            # using active scanning).
+            # That is why discovery_stopped_callback is not added to self._discovery_stopped_callbacks
+            # at this point, as org.bluez.Adapter1.Discovering status is not relevant for passive
+            # scanning.
+
+            # If advertisement monitor is released before the scanning is stopped, it means that the
+            # kernel does not support passive scanning and error was returned when trying to execute
+            # MGMT command "Add Adv Patterns Monitor" (see https://github.com/hbldh/bleak/issues/1136).
+            def advertisement_monitor_released() -> None:
+                # FIXME: This is not actually raised
```

Comment:
I've again hit a roadblock and would be happy for some ideas on how to solve it. So, there are many reasons why active discovery could be (prematurely) stopped (`Discovering` set to `False`), but only a specific case when Advertising Monitor is released.
I want to use this fact to detect that the passive scanning is not supported, instead of just informing the user that something stopped the discovery (e.g. using HCI injection or mgmt commands) and it should be restarted (which would not help in this case).
Trying to achieve that `BleakNoPassiveScanError` would be raised (preferably in the `start()` method), I noticed that I came really close to the previous solution, where I "processed the D-Bus events" after exporting the Advertising Manager :thinking: